src/components/velocity.js

```diff
@@ -0,0 +1,25 @@
+var registerComponent = require('../core/component').registerComponent;
+
+/**
+ * Velocity component.
+ *
+ * Updates object's position at each tick.
+ */
+module.exports = registerComponent('velocity', {
+  dependencies: ['position'],
```

Comment:
ah! ok thanks. 

qq on that – the first time I call `el.getAttribute('position')` I'm getting `undefined`, even though it's a default dependency. But `el.getComputedAttribute('position')` returns `{x:0,y:0,z:0}`.  After the position has been set once, either of those works fine. Is that a bug? Or is `getComputedAttribute` just the thing I should use?